src/Utilities/Utils.res

```diff
@@ -1375,3 +1375,5 @@ let getFirstAndLastNameFromFullName = fullName => {
 
   (firstName, lastNameJson)
 }
+
+let checkIsTestCardWildcard = val => ["1111222233334444"]->Array.includes(val)
```

Comment:
1. Why are we adding specific card numbers directly. Instead can we add a check that if a card passes the luhn algorithm, consider it as a wild card and pass the validation for that card?
2. We will not have cardBrands for these card numbers, will it be an issue if we do not pass the card brand in the confirm request?
3. This might be a good chance to check for which all card numbers our regex validation is not working. @Sanskar2001

Reply:
@ArushKapoorJuspay cards scheme range check is a norm and we should not allow the cards which don't have a valid scheme. The exception should be made for some wild cards only.

Reply:
for 2nd one there is no issue if we do not pass card_network for such cards